ui/src/dataExplorer/components/SaveAsCellForm.tsx

```diff
@@ -22,7 +22,8 @@ import {
 } from 'src/dashboards/constants'
 
 // Actions
-import {getDashboards, createCellWithView} from 'src/dashboards/actions/thunks'
+import {getDashboards} from 'src/dashboards/actions/thunks'
```

Comment:
do you have this thunks vs creators convention written in a style guide-ish something somewhere? might be helpful to share around again, because i forget

Reply:
its currently just in the RFC.  But yea, I just find it cleaner to separate the action creators from the thunks.